utils/guidelines_xslt/odd2html/guidelines.xsl

```diff
@@ -463,6 +462,15 @@
         <span class="mentioned"><xsl:apply-templates select="node()" mode="#current"/></span>
     </xsl:template>
     
+    <xd:doc>
+        <xd:desc>
+            <xd:p>Quoted content</xd:p>
+        </xd:desc>
+    </xd:doc>
+    <xsl:template match="tei:q|tei:quote" mode="guidelines">
+        <xsl:value-of select="'&quot;'"/><xsl:apply-templates select="node()" mode="#current"/><xsl:value-of select="'&quot;'"/>
```

Comment:
second thought after approving: we could handle this via CSS and then even decide on hos the quotes actually look like with the CSS `quotes:` property what do you think?

Reply:
I thought about this, but I'd say this is actual _content_ and not just _styling_.